src/index.js

```diff
@@ -0,0 +1,80 @@
+//import inquirer
+const inquirer = require("inquirer");
+
+//import classes
+const { Engineer, Manager, Intern } = require("./lib");
+
+// import questions
+const {
+  teamQuestion,
+  loopQuestion,
+  managerQuestions,
+  engineerQuestions,
+  internQuestions,
+} = require("./modules/questions");
+
+//import utils
+const {
+  constructAnswersObject,
+  generateHtml,
+  writeToFile,
+} = require("./modules/utils");
+
+const start = async () => {
+  let inProgress = true;
+
+  const employees = [];
+
+  //get team name
+  const { teamName } = await inquirer.prompt(teamQuestion);
+
+  //get manager answers
+  const { name, id, email, officeNumber } = await inquirer.prompt(
+    managerQuestions
+  );
+  const manager = new Manager({ name, id, email, officeNumber });
+
+  employees.push(manager);
+
+  while (inProgress) {
+    const { role } = await inquirer.prompt(loopQuestion);
+
+    if (role === "Engineer") {
+      //get engineer answers
+      const { name, id, email, github } = await inquirer.prompt(
+        engineerQuestions
+      );
+
+      const engineer = new Engineer({ name, id, email, github });
+
+      employees.push(engineer);
+    }
+
+    if (role === "Intern") {
+      //get intern answers
+      const { name, id, email, school } = await inquirer.prompt(
+        internQuestions
+      );
+      const intern = new Intern({ name, id, email, school });
+
+      employees.push(intern);
+    }
+
+    if (role === "None") {
+      inProgress = false;
+    }
+  }
+
+  //build answers object
+  const answers = constructAnswersObject(teamName, employees);
```

Comment:
I don't think we really need to pass the data through this function `constructAnswersObject`. It's an unnecessary extra function.

You can categorize your employees here directly: 
`const categorizedEmployees = categorizeEmployees(employees);`

and then pass two variables separately to the `generateHtml` function:

`generateHtml(teamName, categorizedEmployees)`

^ we would just need to tweak the `generateHtml` function to then receive two variables.

Reply:
If you REALLY want to have everything in one object, we can do it directly here rather than using that extra function, with our `...` spread operator:

`const newObject = { ... categorizedEmployes, teamName }`